kohese-portal/common/models/Namespace.json

```diff
@@ -16,6 +16,10 @@
       "type": "string",
       "required": false,
       "default": ""
+    },
+    "repositoryId": {
```

Comment:
This seems to be acting like a reference instead of a string.  I would have thought the type would be [ "Repository" ].

Reply:
Should this be a derived attribute?